mongo/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## To be Released
 
+feat(pagination): implement go-handlers 400 errors and return 200 when the page is empty
```

Comment:
Nitpick: the sentence is not really understandable, maybe something like:
```suggestion
feat(pagination): replace BadRequestError by the one from go-handlers to return HTTP 400.
feat(pagination): return HTTP 200 when the page is empty instead of BadRequestError
```
would be better, WDYT?

The idea is: it is a breaking change, we need to be explicit about it.